packages/react/src/components/ExpressiveModal/ExpressiveModal.js

```diff
@@ -20,13 +20,21 @@ const { prefix } = settings;
  * Expressive Modal
  *
  * @param {object} props react proptypes
+ * @param {boolean} props.fullwidth sets whether the modal is fulldiwth or not
```

Comment:
@sannminwin can you also add this new prop to the README?

Reply:
updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/817/commits/d93508d6d14583824606a25a36c762dbe1f5cee4)